src/main/java/com/limechain/sync/warpsync/dto/RuntimeCodeException.java

```diff
@@ -0,0 +1,7 @@
+package com.limechain.sync.warpsync.dto;
+
+public class RuntimeCodeException extends Exception {
+    public RuntimeCodeException(String errorMessage) {
+        super(errorMessage);
+    }
+}
```

Comment:
`Exception` is a checked exception - never extend it. Instead extend `RuntimeException`